src/utils/StateUpdaters.tsx

```diff
@@ -172,25 +272,50 @@ const getCombinedRowVisibility = (index: number): Types.Visibility => {
   return isMarkerVisible;
 };
 
-export const getMarkerIdentifier = (index: number): string => {
-  return Config.componentIds.Marker(index);
-};
-
 const elementIsInViewport = (element: HTMLElement | null): Types.Visibility => {
```

Comment:
Related to "Fixed visibility bug where offscreen markers are labeled visible and vice versa"